R/pairwise-comparisons.R

```diff
@@ -378,3 +380,62 @@ infer_rel_skill_metric <- function(scores) {
 
   return(rel_skill_metric)
 }
+
+
+#' @title Calculate Geometric Mean
+#'
+#' @details
+#' Used in [pairwise_comparison()].
+#'
+#' @param x numeric vector of values for which to calculate the geometric mean
+#' @return the geometric mean of the values in `x`
+#'
+#' @keywords internal
+geom_mean <- function(x) {
```

Comment:
can we call this geometric_mean? My first thought was that this was for ggplot2

Reply:
ah right - good idea

Reply:
see https://github.com/epiforecasts/scoringutils/issues/345